app/controllers/users_controller.rb

```diff
@@ -67,13 +67,14 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_user
-      @user = User.find(params[:id])
-    end
+  # Use callbacks to share common setup or constraints between actions.
```

Comment:
send can be used to access private methods.

public_send calls public methods only.